blog/2024-04-05-translation-key-filtering.md

```diff
@@ -0,0 +1,17 @@
+---
+author: Joost Lekkerkerker
+authorURL: https://github.com/joostlek
+authorImageURL: https://avatars.githubusercontent.com/u/7083755?v=4
+title: "Changing the way translation keys are handled"
+---
+
+In Home Assistant Core 2024.4.2, we are changing the way translation keys are downloaded.
+
+Previously the script would download every translated string.
+With this change, we only download the strings that are verified.
+
+A verified string is a translated string, and it becomes unverified when the original string changes.
+So whenever someone changes something in the original string, the translated string becomes unverified and needs to be reviewed again.
```

Comment:
"something" is a pleonasm. "reviewed" is another Lokalize concept. Suggestions:
> So whenever someone changes the default value of a string (i.e. changes a `strings.json` file), all translations of that key becomes unverified and must be edited or marked as verified in Lokalise.

Reply:
The "default" is English.

> So whenever the English string has been modified (i.e. changes a `strings.json` file), all translations of that key becomes unverified and must be edited or marked as verified in Lokalise.

Reply:
> So whenever the English string is modified (i.e. changes a strings.json file), all translations which refers to the same translation key becomes unverified and must be edited or marked as verified in Lokalise.